commands/copy.js

```diff
@@ -14,12 +14,12 @@ function * run (context, heroku) {
   let resolve = co.wrap(function * (db) {
     if (db.match(/^postgres:\/\//)) {
       let uri = url.parse(db)
-      let name = uri.path ? uri.path.slice(1) : ''
-      let hostname = `${uri.host}:${uri.port || 5432}`
+      let dbname = uri.path ? uri.path.slice(1) : ''
```

Comment:
I didn't really bother trying to look into more, but this kinda doesn't make sense that `uri.path` being potentially null  -- we obviously can't `pg:copy` from/to the database that doesn't have dbname with it.
Let's pretend like we didn't see it for now, though.